app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -130,6 +130,7 @@ func (l *LoadBalancerSyncer) CreateLoadBalancer(ing *v1beta1.Ingress, forceUpdat
 	// Create backend service. This should always be called after the health check since backend service needs to point to the health check.
 	backendServices, beErr := l.bss.EnsureBackendService(l.lbName, ports, healthChecks, namedPorts, igsForBE, forceUpdate)
 	if beErr != nil {
+		fmt.Printf("Error ensuring backend service for %s: %v", l.lbName, beErr)
```

Comment:
is l.lbname the right name to print out here?

Reply:
yup